apps/extension/src/Setup/AccountCreation/Steps/Password/Password.tsx

```diff
@@ -72,7 +70,7 @@ const Password: React.FC<Props> = (props) => {
         </BodyText>
 
         <InputContainer>
-          <Header5>Alias (optional)</Header5>
+          <Header5>Alias</Header5>
```

Comment:
Is alias still optional for non-ledger accounts? I can still create an account without an alias, but not sure if that is expected behaviour.

Reply:
I think we should make it required. The set-up process doesn't require it yet, but elsewhere in the app it is now required.

Reply:
I've updated the validation to make this a required field!